common-types/index.d.ts

```diff
@@ -144,10 +144,12 @@ interface DevPortfolio {
   earliestValidDate: number;
   submissions: DevPortfolioSubmission[];
   readonly uuid: string;
+  lateDeadline?: number;
 }
 
 interface DevPortfolioSubmission {
   member: IdolMember;
   openedPRs: PullRequestSubmission[];
   reviewedPRs: PullRequestSubmission[];
+  isLate?: boolean;
```

Comment:
Will this run into the same issue as #347 where the db can't handle fields that are `undefined`?

Reply:
On the backend, `isLate` is always set [here](https://github.com/cornell-dti/idol/blob/dp/late-submissions/backend/src/API/devPortfolioAPI.ts#L90-L93) so we would never try to set `undefined` into the db.